src/app/drive/components/DriveExplorer/DriveExplorerList/DriveExplorerList.tsx

```diff
@@ -495,10 +495,13 @@ const DriveExplorerList: React.FC<DriveExplorerListProps> = memo((props) => {
           selectedItems={props.selectedItems}
           keyboardShortcuts={['unselectAll', 'selectAll', 'multiselect']}
           keyBoardShortcutActions={{
-            onBackspaceKeyPressed: () =>
-              isSelectedSharedItems
-                ? props.onOpenStopSharingAndMoveToTrashDialog()
-                : moveItemsToTrash(props.selectedItems),
+            onBackspaceKeyPressed: () => {
+              if (props.selectedItems.length === 1) {
+                isSelectedSharedItems
```

Comment:
nit :Add the condition to a variable with descriptive name in order to give it more context

Reply:
In fact this is a bug, because it should be >0 in case you select various items.